cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js

```diff
@@ -29,6 +31,10 @@ server.get('FetchOAuthToken', server.middleware.https, function (req, res, next)
         // store OAuth token expire time in milliseconds, 1000 -> ONE_SECOND
         session.privacy.boltOAuthTokenExpire = response.result.expires_in * 1000
             + new Date().getTime();
+        var currentBasket = BasketMgr.getCurrentBasket();
```

Comment:
might need to consider the use case that currentBasket does not exists. Can we use `basketMgr. getCurrentOrNewBasket()`

Reply:
Make sense!